app.rb

```diff
@@ -0,0 +1,46 @@
+require 'sinatra/base'
+require './lib/peep'
+require './lib/user'
+require './database_connection_setup'
+
+class Chitter < Sinatra::Base
```

Comment:
Generally your controller looks good.  It's not doing any logical heavy lifting and all your routes look pretty concise.

One thing you might want to take a look at is RESTful routing - it's a set of conventions for how we should name routes in a web application and is a really easy wind in terms of making your code look of a professional standard.